### ISSUE TITLE
prepare clients dataset

### ISSUE THREAD

**Issue by xiqxin1** [YikFungTsai/DistFL#1]
Hi

Thanks for sharing your work. Could you tell me how to correctly process the cifar-10 dataset into 20 clients?

Maybe there're additional functions before "train_dataset, test_dataset = get_dataset(args)", could you check it?

BTW, it's difficult to imagine how to process the home_office dataset.
For example, when preparing 5 clients in Art, did you randomly split the folders in Art into 5 groups, and then split train/test sets in each folder?

**Comment by YikFungTsai** [comment]
Sorry, I miss the issue.

The codes in this repository load the pre-partitioned Office-Home dataset into 20 clients. As you said, we randomly partitioned the datasets in the art/product/clipart/real-world domains into 5 clients each. Clients have their own train set and share a test set in each domain.

If you want to load the cifar-10 dataset, you can also follow these rules: for each group (which can be 10) of clients randomly select two classes in the dataset and split the train set into each client while they have the full test set of the corresponding classes.

Thanks for your issue.
